mockserver.js

```diff
@@ -60,11 +61,73 @@ var parse = function (content) {
         }
     });
 
-    body = bodyContent.join('\n');
+    body = applyHooks(bodyContent).join('\n');
```

Comment:
While adding momentjs gives obvious benefits, adding templating engine doesn't solve any problem by itself. There are things that can be done to improve the tool, but I wouldn't go any further without a clear vision, spending some time with hacky prototypes to prove ideas and free time to build it properly